models/activity.py

```diff
@@ -0,0 +1,17 @@
+from utility.helpers import query
+
+
+class Activity():
+
+    def __init__(self):
+        pass
+
+    def new_action(self, action, description, date_time, user_id, contact_id):
```

Comment:
Common method names should follow a consistent pattern to prevent misunderstandings.

Reply:
sure

Reply:
Can I rename the old ones? for example from "register" to "add_user" ?

Reply:
`User.add_user`? Sounds redundant.

Reply:
You are right. I renamed "new action" to "add"